src/routes/search/components/SearchModal.js

```diff
@@ -61,8 +65,13 @@ class SearchModal extends React.Component<PropsType, StateType> {
       .map(category => ({model: category, subCategories: []}))
   }
 
-  onItemPress = (category: {path: string}) => {
+  onItemPress = (category: { path: string }) => {
```

Comment:
Should we try to always use strict props again?

Reply:
We could force this with a linting rule. But there are a lot of places right now where it could be used. 

I think it makes most sense for PropTypes and the state. If we have them there I'm fine.